linkcovery/core/utils.py

```
@@ -95,21 +97,88 @@ def handle_starttag(self, tag, attrs) -> None:
+        Fetched description or empty string on failure
+
+    """
+    if show_spinner:
+        from rich.status import Status
+
+        with Status("📥 Fetching metadata...", console=console):
+            try:
+                async with AsyncClient(
+                    timeout=timeout,
+                    follow_redirects=True,
+                    verify=False,
```

Ruff (0.15.0)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)